src/trusted/service_runtime/sel_ldr.h

```diff
@@ -114,6 +114,12 @@ struct NaClSpringboardInfo {
   uint32_t end_addr;
 };
 
+enum NaClThreadLaunchType {
+  MAIN,
+  FORK,
+  EXEC
+};
```

Comment:
I think to make sure we don't run into naming conflicts later down the line that are hard to diagnose something like 
THREAD_LAUNCH_MAIN
THREAD_LAUNCH_FORK
THREAD_LAUNCH_EXEC
would be better.

Reply:
This was a good idea, I just updated to reflect this.